### PR TITLE
Add autofix to media-feature-range-operator-space-before

### LINKED ISSUE #3580: Add autofix to media-feature-range-operator-space-before
> What is the problem you're trying to solve?

Adds autofix of rules [defined in `stylelint-config-standard`](https://github.com/stylelint/stylelint-config-standard/blob/master/index.js) so that everyone can start using [`stylelint-config-standard`](https://github.com/stylelint/stylelint-config-standard) easily.

> What solution would you like to see?

Add autofix to [media-feature-range-operator-space-before](https://github.com/stylelint/stylelint/tree/master/lib/rules/media-feature-range-operator-space-before)

### PR DESCRIPTION
**Pull request by ota-meshi**
> Which issue, if any, is this issue related to?

closes #3580

> Is there anything in the PR that needs further explanation?

if use the `--fix` option, to remove whitespace at the end of each line.

ex.

* option: `always`

    code:
    ```css
    @media (width<600px) {}
    ```
    fixed:
    ```css
    @media (width <600px) {}
    ```

* option: `never`

    code:
    ```css
    @media (width < 600px) {}
    ```

    fixed:
    ```css
    @media (width< 600px) {}
    ```

### CODE REVIEW

File: lib/rules/media-feature-range-operator-space-before/index.js [hunk 1]
```diff
@@ -26,26 +26,54 @@ const rule = function(expectation) {
...
-        index: match.index + 1,
-        err: m => {
-          report({
-            message: m,
-            node,
-            index: match.index + atRuleParamIndex(node),
-            result,
-            ruleName
+      if (fixOperatorIndices.length) {
+        let params = atRule.raws.params
```
[the remark this answers is not in the document]
Comment (ota-meshi) [reply lib/rules/media-feature-range-operator-space-before/index.js]: However, in fact, it seems to exist.

You can try it below.
http://astexplorer.net/#/gist/3596e8f68b84c0dc6d9d69866d5fa505/7716a73333c0e9b54eed5997ff4887eec16ab87c

[the remark this answers is not in the document]
Comment (boccob) [reply lib/rules/media-feature-range-operator-space-before/index.js]: Is there any difference between getting / setting `node.params` and `node.raws.params.raw`?  
If there is no difference I think it'd be better to use documented parameters.

File: lib/rules/media-feature-range-operator-space-before/index.js [hunk 2]
```diff
@@ -26,26 +26,54 @@ const rule = function(expectation) {
     }
 
     root.walkAtRules(/^media$/i, atRule => {
+      const fixOperatorIndices = [];
       findMediaOperator(atRule, checkBeforeOperator);
```
[the remark this answers is not in the document]
Comment (ota-meshi) [reply lib/rules/media-feature-range-operator-space-before/index.js]: I think that is good. I will fix it the next time I can work.

File: lib/rules/media-feature-range-operator-space-before/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ota-meshi) [reply lib/rules/media-feature-range-operator-space-before/index.js]: `node.raws.params.raw` to be create if comments get in. In this case, `node.params` will not contain comment data.
I tried finding but I do not know the existence of the document yet. . .

[the remark this answers is not in the document]
Comment (boccob) [reply lib/rules/media-feature-range-operator-space-before/index.js]: Got it. Thank you!

**Comment by ota-meshi** [comment]
@boccob
I changed the content pointed by your review.

**Comment by jeddy3** [comment]
> -   Added: `media-feature-range-operator-space-before` autofix ([#3618](https://github.com/stylelint/stylelint/pull/3618)).

**Pull request merged**